executor/include/executor/trait/can_require.hpp

```diff
@@ -14,24 +14,32 @@
 
 namespace executor {
 
+namespace detail {
+template <typename Executor, typename Property>
+using contains_property = std::is_same<
+    std::remove_const_t<decltype(Property::template static_query_v<Executor>)>,
```

Comment:
This `std::remove_const_t` should not be required, because in general you don't return neither pointers nor references from `Property::template static_query_v<Executor>`, but entirely new objects. What made you add it?

Reply:
`Property::template static_query_v<Executor>` is a `constexpr` so it returns a const object